src/main/java/org/fundacionjala/pivotal/core/webdrivers/DriverManager.java

```diff
@@ -4,66 +4,73 @@
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.support.ui.WebDriverWait;
 
+import java.util.concurrent.TimeUnit;
+
 /**
  * Class Web util Driver.
  */
 public final class DriverManager {
 
-    /**
-     * WebDriver driver declared.
-     */
+    private static final int EXPLICIT_TIME = 60;
+    private static DriverManager driverManager;
     private WebDriver driver;
-
-    /**
-     * WebDriverWait driverWait declared.
-     */
     private WebDriverWait driverWait;
 
     /**
-     * DriverManager instance declared.
-     */
-    private static DriverManager instance;
-
-    /**
-     * Method Web util Driver.
-     * Use Singleton Pattern.
+     * Constructor, private to apply singleton pattern.
      */
     private DriverManager() {
-        final int explicitWaitTimeOut = 30;
         DriverType driverType = DriverType.valueOf(PropertiesConfig.getInstance().getBrowser());
         driver = DriverFactory.getDriverManager(driverType);
-        driver.navigate().to("https://www.pivotaltracker.com/signin");
-        driverWait = new WebDriverWait(driver, explicitWaitTimeOut);
+        //webDriver.navigate().to("https://www.pivotaltracker.com/signin");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)